pcsx2/gui/MainFrame.cpp

```diff
@@ -533,17 +535,25 @@ void MainEmuFrame::CreateCaptureMenu()
 	m_submenuScreenshot.Append(MenuId_Capture_Screenshot_Screenshot_As, _("Screenshot As..."));
 }
 
-void MainEmuFrame::CreateRecordMenu()
+void MainEmuFrame::CreateInputRecordingMenu()
 {
 #ifndef DISABLE_RECORDING
 	m_menuRecording.Append(MenuId_Recording_New, _("New"), _("Create a new input recording."))->Enable(false);
 	m_menuRecording.Append(MenuId_Recording_Stop, _("Stop"), _("Stop the active input recording."))->Enable(false);
 	m_menuRecording.Append(MenuId_Recording_Play, _("Play"), _("Playback an existing input recording."))->Enable(false);
 	m_menuRecording.AppendSeparator();
+
+	m_menuRecording.Append(MenuId_Recording_Settings, _("Settings"), &m_submenu_recording_settings);
```

Comment:
I don't think a new sub menu is needed for just 1 option.

Reply:
This is true, however I have more planned settings to go into this sub-menu, so I went in this direction to get things ready for that.

Reply:
Makes sense, it's fine then.